include/quill_varying_impl.h

```diff
@@ -35,7 +35,7 @@ inline Varying2D operator-(Varying2D a, float u)     { return Varying2D(a.x - u,
 inline Varying2D operator-(float u, Varying2D a)     { return Varying2D(u - a.x,   u - a.y);   }
 inline Varying2D operator*(Varying2D a, float u)     { return Varying2D(a.x * u,   a.y * u);   }
 inline Varying2D operator*(float u, Varying2D a)     { return Varying2D(u * a.x,   u * a.y);   }
-inline Varying2D operator/(Varying2D a, float u)     { return Varying2D(a.x / u,   a.y / u);   }
+inline Varying2D operator/(Varying2D a, float u)     { if (Q_UNLIKELY(u == 0)) return Varying2D(0, 0); return Varying2D(a.x / u,   a.y / u);   }
```

Comment:
We cannot make use of Q_UNLIKELY here, this is non-Qt code... Also, it doesn't make sense to solve just this case, and not the one above it, or the Varying3D case?

Reply:
was a very quick fix, just fixed the issues ubsan highlighted while doing some testing during the weekend.

The problem is that the best name for our own macro is Q_UNLIKELY considering the name of the library being quill. :-P